src/Microsoft.DotNet.Wpf/src/DirectWriteForwarder/CPP/DWriteWrapper/TextAnalyzer.cpp

```diff
@@ -276,19 +276,6 @@ namespace MS { namespace Internal { namespace Text { namespace TextInterface
         }
     }
 
-// Warning 4714 (__forceinline function not inlined)
```

Comment:
I'm not so sure about suddenly inlining a set of function that were not inlined previously.  I understand the original reasoning for them not being inlined has gone away, but we need to be 100% certain there are no adverse affects from doing this.

Can we hold off on this, file an issue and do a more thorough examination of these changes with a larger test pass later?

Reply:
I think I misunderstood this, it's just disabling a warning, not enabling inlining, resolving.